pkgs/applications/virtualization/nvidia-docker/default.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv, lib, fetchFromGitHub, callPackage, makeWrapper, buildGoPackage, runc
+, libelf, libcap, libseccomp , nvidia_x11, glibc }:
+
+with lib; let
+
+  libnvidia-container = callPackage (import ./libnvc.nix) {
+    inherit nvidia_x11;
+  };
+
+  nvidia-container-runtime = fetchFromGitHub {
+    owner = "NVIDIA";
+    repo = "nvidia-container-runtime";
+    rev = "95836f333576730bf0cc5f3e6580a30495034926";
```

Comment:
Was it possible to stick to released versions? https://github.com/NVIDIA/nvidia-container-runtime/releases